tests/python_package_test/test_engine.py

```diff
@@ -24,6 +24,28 @@ def multi_logloss(y_true, y_pred):
     return np.mean([-math.log(y_pred[i][y]) for i, y in enumerate(y_true)])
 
 
+def test_constant_features(y_true, expected_pred, more_params):
```

Comment:
@StrikerRUS  how to stop the test on this function?

Reply:
@guolinke Sorry, what do you mean under "to stop the test"? A breakpoint? Or skip all further functions?

If latter, you can simply rename all unneeded functions so that they don't have `test_` prefix in their names.

Reply:
Thanks, I fixed that by default parameters.

Reply:
Oh, I see. However, such approach adds dummy record into `pytest`'s overall number of tests.